ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -417,9 +426,14 @@ snpe_subplugin::invoke (const GstTensorMemory *input, GstTensorMemory *output)
 
   /* Configure inputs */
   for (unsigned int i = 0; i < inputInfo.num_tensors; ++i) {
-    float *finput = (float *)input[i].data;
     size_t fsize = input_tensors[i].get ()->getSize ();
-    std::copy (finput, finput + fsize, input_tensors[i].get ()->begin ());
+    if (is_input_type_uint8) {
+      uint8_t *inbuf = (uint8_t *) input[i].data;
```

Comment:
use input[i].data in std:copy().
if failed, use uint8_t *inbuf = (uint8_t *) input[i].data; to copy data.